src/openforms/formio/tests/validation/test_file.py

```diff
@@ -29,6 +35,19 @@
 }
 
 
+def validate_formio_data(
+    component: FileComponent, data: JSONValue, session_uploads: list[str] = []
+) -> tuple[bool, ReturnDict]:
+    request = APIRequestFactory().put("/irrelevant")
+
+    # Add a session to the request:
+    middleware = SessionMiddleware(lambda x: None)  # type: ignore
+    middleware.process_request(request)
+    request.session[UPLOADS_SESSION_KEY] = [str(uuid) for uuid in session_uploads]
```

Comment:
I took a different approach in other tests: https://github.com/maykinmedia/mozilla-django-oidc-db/blob/master/tests/conftest.py#L120